incubator/hnc/pkg/validators/hierarchy.go

```diff
@@ -33,13 +36,16 @@ const (
 type Hierarchy struct {
 	Log     logr.Logger
 	Forest  *forest.Forest
-	authz   authzClient
+	server  serverClient
```

Comment:
Maybe still call it `client` and call `v.getServerChecks()` `v.checkWithServer()`?